docs/gallery-submission.md

```diff
@@ -149,28 +149,52 @@ A short list of email addresses to give access to the Beta feedback. Should be i
 Beta Process and Preparing Your Go-to-Market Materials
```

Comment:
This is now out of sync with the pdf version linked above.  Do we need both representations?

Reply:
I'll update that as well--thanks